<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1472</width>
    <height>989</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelMainText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>621</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HP Simplified</family>
      <pointsize>46</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Syötä kortti</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMainImage">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnectionText">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>800</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;connectionText&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1472</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
